csrc/scheduler/mma_utils.cpp

```diff
@@ -831,10 +831,6 @@ void scheduleLdMatrix(TensorView* tv, MmaOptions options) {
   //   if tv is immediate output of ldmatrix
```

Comment:
I think this function needs to be refactored, but it is not the scope of this PR.